generic/piranha-tree-sitter/src/cleanup_rules/java/edges.toml

```diff
@@ -0,0 +1,71 @@
+# The edges in this file specify the the flow between the rules
+
+[[edges]]
+from = "Returns boolean"
+scope = "Parent"
+to = ["Boolean literal cleanup"]
```

Comment:
also, can we define a structure for these names? verb_type? 
`cleanup_boolean_literal`, `simplify_boolean_expression`, `return_boolean`,

Reply:
Addressed.